vignettes/secuTrialR-package-vignette.Rmd

```diff
@@ -727,22 +727,91 @@ forms: a_form_name"
 )
 ```
 
+# Merging forms in the `secuTrialdata` object
+
+Naturally, you will sometimes need to merge/join some of the data from the individual form data stored
+in your `secuTrialdata` object. To achieve this you can use the `merge()` function.
+A common task may be to merge repetition form data to its parent form.
+In our case `esurgeries` can be naively merged with `baseline` via the `mnpdocid`:
+
+\vspace{5pt}
+```{r}
+bl_surg <- merge(x = ctu05_data$baseline, y = ctu05_data$esurgeries, by = "mnpdocid")
+```
+
+Please note, that the merging can cause duplication of data if the ids that the merge is directed by are not unique.
```

Comment:
I think it would be "Please note, that merging can cause" (no the)... wouldn't it?

Reply:
correct